### PR TITLE
Improve logging for VA integrations

### PR DESCRIPTION
**Pull request by vibhanshub**

### CODE REVIEW

File: integrations/va-auth0/src/index.tsx [hunk 1]
```diff
@@ -225,9 +225,19 @@ const handleFetchEvent: FetchEventCallback<Auth0RuntimeContext> = async (request
...
-                        });
+                        logger.debug(
+                            `Did not receive access token. Error: ${(resp && resp.error) || ''} ${
+                                (resp && resp.error_description) || ''
+                            }`
+                        );
+                        return new Response(
+                            `Error: No Access Token found in response from Auth0 ${
+                                (resp && resp.error) || ''
+                            } ${(resp && resp.error_description) || ''}`,
```
Comment (scazan) [inline integrations/va-auth0/src/index.tsx:236]: Have you verified that error.description won't possibly leak any keys into the logs?

Comment (vibhanshub) [reply integrations/va-auth0/src/index.tsx:236]: These errors are returned by the auth providers, so they're unlikely to contain any keys. However, there is no way to confirm that. At least I can't think of a way to confirm that

Comment (taranvohra) [reply integrations/va-auth0/src/index.tsx:236]: I agree with @scazan. We should not surface errors directly as it could contain sensitive information.

**Review by taranvohra — DISMISSED** [review]

File: integrations/va-auth0/src/index.tsx (hunk 1 above)
Comment (vibhanshub) [reply integrations/va-auth0/src/index.tsx:236]: @taranvohra but is it fine to log them? And can we surface just response.error to the user (and not response.error_description)

Comment (taranvohra) [reply integrations/va-auth0/src/index.tsx:236]: Logs are fine because only the integration owner can see them.

Comment (vibhanshub) [reply integrations/va-auth0/src/index.tsx]: Fixed!

**Review by scazan — APPROVED** [review]

**Pull request merged**